PyMca5/tests/ut/enable_pymca_import.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+"""enable_pymca_import.py
+
+    Enable tests to import PyMca5 modules.
+
+    By default PyMca5 cannot be imported when called from a source directory.
+    Import _this_ module before importing PyMca5 or one of its submodules
+    to bypass that limitation, or override this behavior
+    by setting the environment variable ALLOW_PYMCA_IMPORT.
+
+    Examples:
+
+    ```python
+    import enable_pymca_import  # noqa: F401
+
+    # These imports will now succeed!
+    import PyMca5
+    from PyMca5.PyMcaCore.DataObject import DataObject
+    ```
+
+    ```shell
+    # These tests will fail to import PyMca5
+    ALLOW_PYMCA_IMPORT=0 python -m pytest
+    ALLOW_PYMCA_IMPORT=false python -m pytest
+    ALLOW_PYMCA_IMPORT=FALSE python -m pytest
+
+    # These tests will successfully import PyMca5
+    ALLOW_PYMCA_IMPORT=1 python -m pytest
+    ALLOW_PYMCA_IMPORT=true python -m pytest
+    ALLOW_PYMCA_IMPORT=TRUE python -m pytest
+    ```
+
+    Tests that are nested within the PyMca5 module must have a parent directory
+    that does NOT contain an __init__.py file. Otherwise, pytest will attempt
+    to import PyMca5 before the tests are collected.
+"""
+
+import os
+
+
+def enforce_bool(value) -> bool:
```

Comment:
Duplicated from above?

Reply:
See earlier [note](https://github.com/NSLS2/pymca/pull/12#discussion_r1759666204).

Please disabuse me of this notion though.  I welcome good excuses to de-duplicate code.

Reply:
Probably the right thing to do would be to extract these functions into a separately installed package (which gets its own testing).

Reply:
Will submit this as a issue to resolve later.

Reply:
https://github.com/NSLS2/pymca/issues/13